.github/actions/custom-actions/aselo_production_custom/action.yml

```diff
@@ -22,11 +22,11 @@ runs:
   using: "composite"
   steps:
     # Line environment variables
-    - name: Set helpline Line Flex Flow SID
+    - name: Set helpline Line Studio Flow SID
       uses: "marvinpinto/action-inject-ssm-secrets@latest"
       with:
-        ssm_parameter: "PROD_TWILIO_AS_LINE_FLEX_FLOW_SID"
-        env_variable_name: "LINE_FLEX_FLOW_SID"
+        ssm_parameter: "/production/twilio/${{inputs.account-sid}}/line_studio_flow_sid"
```

Comment:
Thats good to know its is Studio id we need 😅 

I just checked AWS parameter store I don't see a value for "/production/twilio/${{inputs.account-sid}}/line_studio_flow_sid". I am wondering if you will be able to input this